lib/hammer_cli/ssloptions.rb

```diff
@@ -0,0 +1,27 @@
+module HammerCLI
+  def self.get_ssl_options
+    settings = HammerCLI::Settings
+    logger = Logging.logger['SSLoptions']
+    ssl_options = {}
+    for sslopt in [:ssl_ca_file, :ssl_ca_path] do
+      ssloptval = settings.get(:_params, sslopt) || settings.get(:ssl, sslopt)
+      ssl_options[sslopt] = ssloptval if ssloptval
+    end
+    ssl_client_cert = settings.get(:_params, :ssl_client_cert) || settings.get(:ssl, :ssl_client_cert)
+    ssl_client_key = settings.get(:_params, :ssl_client_key) || settings.get(:ssl, :ssl_client_key)
+    if ssl_client_cert && ssl_client_key
+      ssl_options[:ssl_client_cert] = OpenSSL::X509::Certificate.new(File.read(ssl_client_cert))
```

Comment:
Please catch exception for file read errors to provide meaningful message. Non-existent file currently makes hammer to fail:
```
hammer --ssl_client_key ~/unknown/key --ssl_client_cert ~/unknown/cert  user list
Error: No such file or directory @ rb_sysopen - /root/unknown/cert
Warning: An error occured while loading module hammer_cli_foreman
Warning: An error occured while loading module hammer_cli_foreman_tasks
Error: No such file or directory @ rb_sysopen - /root/unknown/cert
Warning: An error occured while loading module hammer_cli_katello
Error: No such sub-command 'user'

See: 'hammer --help'
```